pkgs/applications/misc/clipmenu/default.nix

```diff
@@ -4,21 +4,22 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "clipmenu";
-  version = "6.0.1";
+  version = "6.1.0";
 
   src = fetchFromGitHub {
     owner  = "cdown";
     repo   = "clipmenu";
     rev    = version;
-    sha256 = "0053j4i14lz5m2bzc5sch5id5ilr1bl196mp8fp0q8x74w3vavs9";
+    sha256 = "0ddj5xcwrdb2qvrndvhv8j6swcqc8dvv5i00pqk35rfk5mrl4hwv";
   };
 
-  buildInputs = [ makeWrapper ];
+  postPatch = ''
+    substituteInPlace ./Makefile --replace /usr "$out"
+  '';
 
-  installPhase = ''
-    mkdir -p $out/bin
-    cp clipdel clipmenu clipmenud $out/bin
+  buildInputs = [ makeWrapper xsel clipnotify ];
```

Comment:
why was `xsel` and `clipnotify` added to `buildInputs`? I don't think these need to be buildInputs since they are included in the wrapper runtime paths?

Reply:
The upstream Makefile requires them specifically, although I don't know why.

```make
REQUIRED_BINS := xsel clipnotify
```

Reply:
ok this is probably fine then

Reply:
Technically this needs to be in `nativeBuildInputs` because `command -v` checks PATH. Otherwise it would break when cross-compiling.